src/confluent_kafka/cimpl.pyi

```diff
@@ -0,0 +1,339 @@
+from concurrent.futures import Future
```

Comment:
This file is mostly done by `stubgen`, with some edits where stuff called things and it didn't match up. Note the heavy use of `object` as a "we don't know what this type is" marker, which _mostly_ should make mypy whinge when it needs more type info and so help us fix it (v.s. using Any)